src/modules/claims/claims.types.ts

```diff
@@ -42,6 +42,35 @@ export interface Claim {
     namespace: string;
 }
 
+export const isIssuedOnchain = (
```

Comment:
This name makes more sense to me but let me know what you think
```suggestion
export const readyToBeRegisteredOnchain = (
```

Reply:
Make sense. May be we should also instead of Claim have specialized OnChainClaim and OffchainClaim